mlflow/store/tracking/rest_store.py

```diff
@@ -646,7 +655,12 @@ def create_assessment(self, assessment: Assessment) -> Assessment:
                 assessment.trace_id, is_databricks=is_databricks
             ),
         )
-        return Assessment.from_proto(response_proto.assessment)
+        created_assessment = Assessment.from_proto(response_proto.assessment)
+        
+        # Also send to delta archival events table if configured
+        self._maybe_log_assessment_to_delta_archival(created_assessment)
```

Comment:
I think delta archival of assessments should always be handled by the MLflow backend, rather than the client (i.e. the same way that tags, metadata updates, etc.) are handled by the backend. The backend will need to take care of this in order to support assessments from the UI & direct REST API calls anyway.

Let me know if I'm misinterpreting or if this is outdated code, but I think we should remove it.